fizzBuzz.js

```diff
@@ -0,0 +1,36 @@
+function FizzBuzz(word1, word2) {
+	
+	this.word1 = word1;
+	this.word2 = word2;
+	this.word3 = word1 + word2;
+
+	this.read = function(num1, num2) {
+		this.begin = num1;
+		this.end = num2;
+	};
+	
+	this.makeFizz = function() {
```

Comment:
This could be a private function rather than a method on the FizzBuzz object; it's not called from outside of the context of the FizzBuzz instance, and if it were, it may not act as expected.